packages/ui-components/src/components/tag-alarm/tag-alarm.scss

```diff
@@ -0,0 +1,109 @@
+@use 'sass:map';
+
+@import '../../assets/styles/globals';
+
+$icon-sizes: (
+	xxsmall: 12px,
+	xsmall: 16px,
+	small: 18px,
+	normal: 24px,
+	large: 28px
+);
+
+$alarms-severity-colors: (
+	'1': #a71300,
```

Comment:
this color should be one of the colors (or variant) of our color palette.

Reply:
talk with design people to confirm this color because this color doesn't exists in our palette. Probably should be a `kv-color('error', 'dark') = #c61600`

Reply:
![image](https://user-images.githubusercontent.com/6940723/212319246-e513315a-34a2-4819-9ff0-09a0318575cc.png)

Reply:
This color does not exist yet in our color schemes. I believe all the colors for this alarms were "added" recently by the design team with the asset table and then, inside MAPS project, we extended its use in all project for alarms.